### ISSUE TITLE
Edge on Linux support

### ISSUE THREAD

**Issue by dministro** [rosolko/WebDriverManager.Net#196]
_WebDriverManager version: 2.13.0_

`WebDriverManager` with `EdgeConfig` throws `PlatformNotSupportedException` on Linux.

Please add Linux support for edgedriver.

**Comment by bonigarcia** [comment]
It seems the driver for Edge in Linux is not managed with WebDriverManager.Net version 2.14.1. The following tests:

```
        [TestMethod]
        public void EdgeSession()
        {
            new DriverManager().SetUpDriver(new EdgeConfig());

            var driver = new EdgeDriver();

            driver.Quit();
        }
```

... produces the following error on (Ubuntu 20.04.4):

```
  Failed EdgeSession [148 ms]
  Error Message:
   Test method SeleniumDocs.GettingStarted.InstallDriversTest.EdgeSession threw exception: 
OpenQA.Selenium.DriverServiceNotFoundException: The msedgedriver file does not exist in the current directory or in a directory on the PATH environment variable. The driver can be downloaded at https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/.
  Stack Trace:
      at OpenQA.Selenium.DriverService.FindDriverServiceExecutable(String executableName, Uri downloadUrl)
   at OpenQA.Selenium.Edge.EdgeDriverService.CreateDefaultService()
   at OpenQA.Selenium.Edge.EdgeDriver..ctor(EdgeOptions options)
   at OpenQA.Selenium.Edge.EdgeDriver..ctor()
   at SeleniumDocs.GettingStarted.InstallDriversTest.EdgeSession() in /home/boni/dev/seleniumhq.github.io/examples/dotnet/SeleniumDocs/GettingStarted/InstallDriversTest.cs:line 29
```

**Comment by Piedone** [comment]
We're running Edge tests under Ubuntu too (like [this GHA run](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/runs/7792443977?check_suite_focus=true), and the [test results here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/runs/7792752778?check_suite_focus=true) - look for `BasicVisualVerificationTests`). Check out our workarounds for Edge [in this class](https://github.com/Lombiq/UI-Testing-Toolbox/blob/d11b480ec9cd7c53c4e891efe73d1ade711f712a/Lombiq.Tests.UI/Services/WebDriverFactory.cs), especially [here](https://github.com/Lombiq/UI-Testing-Toolbox/blob/d11b480ec9cd7c53c4e891efe73d1ade711f712a/Lombiq.Tests.UI/Services/WebDriverFactory.cs#L208).

**Comment by Piedone** [comment]
Didn't you address this with https://github.com/rosolko/WebDriverManager.Net/commit/a88cd6cad17dda18e535554dc6a598d651a0fe4b, @rosolko?

**Comment by rosolko** [comment]
@Piedone Yes, I did, thanks!

**Comment by rosolko** [comment]
Implemented in 2.15.0 version

**Comment by dministro** [comment]
Hi @rosolko ,

Thank you!
